pkg/ingestor/puller/blob/blob.go

```diff
@@ -14,10 +15,13 @@ import (
 	"github.com/DataDog/KubeHound/pkg/ingestor/puller"
 	"github.com/DataDog/KubeHound/pkg/telemetry/log"
 	"github.com/DataDog/KubeHound/pkg/telemetry/span"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
```

Comment:
This SDK is deprecated, I think it doesn't makes sense for us (as far as I can see, there's no other import of the v1), we should use aws-sdk-go-v2 and this would also remove, IIRC the need for providing the region.

Reply:
+1, at least the AWS SDK can infer the region from the environment (`AWS_REGION`)